PhotoGether/DataLayer/PhotoGetherData/PhotoGetherData/ServiceImpl/WebRTCServiceImpl.swift

```diff
@@ -42,7 +43,11 @@ public final class WebRTCServiceImpl: NSObject, WebRTCService {
         let audioConfig = RTCAudioSessionConfiguration.webRTC()
         audioConfig.category = AVAudioSession.Category.playAndRecord.rawValue
         audioConfig.mode = AVAudioSession.Mode.voiceChat.rawValue
-        audioConfig.categoryOptions = [.defaultToSpeaker]
+        audioConfig.categoryOptions = [
+            .defaultToSpeaker,
+            .allowBluetooth,
+            .allowAirPlay
+        ]
```

Comment:
```suggestion
        audioConfig.categoryOptions = [
            .defaultToSpeaker, // 하단 스피커를 기본으로 설정
            .allowBluetooth, // 블루투스 기기의 음성 입출력 지원
            .allowAirPlay // AirPlay를 통해 연결된 다른 기기로 음성 출력 지원
        ]
```
이렇게 주석 추가해놓는거 어떨까요? 나중에 또 이거 뭐였지 할 수도 있을 것 같아요.

Reply:
[2d2569c](https://github.com/boostcampwm-2024/iOS04-PhotoGether/pull/173/commits/2d2569c7bb9cbaf46371676a4cdefc0060150e13) Suggestion 냠냠